lib/yaml_db/serialization_helper.rb

```diff
@@ -195,7 +196,17 @@ def self.table_record_count(table)
         ActiveRecord::Base.connection.select_one("SELECT COUNT(*) FROM #{Utils.quote_table(table)}").values.first.to_i
       end
 
-    end
+      # Just return the first column as sort key unless both columns are "id
+      # columns", in which case it joins them for ordering
+      def self.sort_key(table)
+        key, second_column = table_column_names(table)
+
+        # If this looks like a standard HABTM join table, add the second "id
+        # column" of the join table for ordering
```

Comment:
Given how small this method is, these two comments (this and the one before the method) are probably redundant with each other.